### ISSUE TITLE
A strange behaviour while training

### ISSUE THREAD

**Issue by Bulld0zzer** [ibab/tensorflow-wavenet#302]
Dear all,

This post is not really an issue! 

However, you have experience with your model and this is the reason why I write this here.
For the task of music generation I used your implementation as it is without any hyper-parameter tuning. I trained the model for about a month. Then I decided to test transfer-learning and trained on a similar genre but totally different audio files.
I encountered the following behavior:
![loss_over_steps](https://user-images.githubusercontent.com/15771726/32594089-da862ae6-c52a-11e7-8a30-aef7ea5e4f34.PNG)

An epoch was reached every 6750 steps. Additionally, I set the silence_threshold to 0.01 (from 0.3). However, I did not expect a behavior like shown above. Unfortunately, I do not have any information about the gradients or histograms of the weights.

Do you have similar experience or any suggestions how to prevent this?

Thanks for any answer,
Daniel
